api/v1alpha4/common_types.go

```diff
@@ -99,15 +99,6 @@ var (
 	ZeroDuration = metav1.Duration{}
 )
 
-const (
-	// MachineNodeNameIndex is used by the Machine Controller to index Machines by Node name, and add a watch on Nodes.
-	MachineNodeNameIndex = "status.nodeRef.name"
```

Comment:
Seems this field was here before v1alpha4, and should we kept

https://github.com/kubernetes-sigs/cluster-api/blob/v0.4.0-rc.0/api/v1alpha4/common_types.go#L103-L104

Reply:
Let's mark it for deprecated in favor of the one under index.

Reply:
>Let's mark it for deprecated in favor of the one under index.

Does this mean adding a comment on top of the constant stating that it is going to be deprecated or is there more to do?

Reply:
Yes, something like:

```
Deprecated: Use .... instead.
```